server/src/remote.rs

```diff
@@ -508,6 +509,23 @@ impl FlightService for FlightServiceHandler {
 
             // Confirm the table was created.
             Ok(Response::new(Box::pin(stream::empty())))
+        } else if action.r#type == "FlushEdge" {
```

Comment:
I personally think that `FlushEdge` makes it clear that the entire edge is flushed of all data.

Reply:
Alternatively, since the [integration testing](https://github.com/ModelarData/ModelarDB-RS/pull/48) branch introduces an action endpoint called `FlushMemory` that does the same as the first part of this action, we could also change this action to something like `TransferAll` and have it only transfer compressed data.

Then a user of the API could simply call these two endpoints one after the other to achieve the same functionality.

Reply:
Personally I would be surprised if an action named `TransferAll` does not transfer the compressed data currently in memory. Let us just stick to `FlushMemory` and `FlushEdge` with their current implementation for now to keep the naming consistent.